README.md

```diff
@@ -172,6 +173,9 @@ To create a custom push provider see [the following section][custom-push-provide
 | Retry register after changing [ANDROID_ID][android-id] |   -   |   -   |          -          |      +      |
 
 
+## Sample
+
+Look through OPFPush library in use in our [sample application][sample].
```

Comment:
I think it would be better to say smth like this:
*Take a look at the usage of the OPFPush library in our [sample application][sample].*